aspnetcore/performance/caching/middleware.md

```diff
@@ -1,15 +1,104 @@
 ---
-title: 🔧 Response caching middleware
+title: Response Caching Middleware | Microsoft Docs
 author: rick-anderson
 ms.author: riande
```

Comment:
They have it setup that Rick is the responsible one ... freeing us to just be 🏈 kids ⚽️. We go on the author byline.